src/subdomains/generic/kyc/controllers/kyc.controller.ts

```diff
@@ -101,6 +108,18 @@ export class KycController {
     return this.kycService.getOrCreateStep(code, ip, stepName, stepType, sequence ? +sequence : undefined);
   }
 
+  @Post('transfer')
+  @UseGuards(AuthGuard(), new RoleGuard(UserRole.ACCOUNT))
+  async addKycClient(@GetJwt() jwt: JwtPayload, @Query('client') walletName: string) {
+    return this.kycService.addKycClient(jwt.account,walletName);
+  }
+
+  @Delete('transfer')
+  @UseGuards(AuthGuard(), new RoleGuard(UserRole.ACCOUNT))
+  async removeKycClient(@GetJwt() jwt: JwtPayload, @Query('client') walletName: string) {
+    return this.kycService.removeKycClient(jwt.account, walletName);
+  }
```

Comment:
Use the KYC code (`@Headers(CodeHeaderName) code: string`), no JWT authentication (see above)